app/models/temp_site.rb

```diff
@@ -0,0 +1,31 @@
+class TempSite < ApplicationRecord
```

Comment:
Wouldn't subclassing `Site` be DRYer?

Reply:
I wonder if although now might seem clunkier, copying things will make it easier for when we do switch the site/temp site model around and then deprecate site. If we do follow a pattern more rejigging might be involved in the end, let me know your thoughts

Reply:
@cwrw Yes I see your point, I'm wary of the amount of code being copied here, and if that might lead to additional work keeping things in sync. If this isn't likely to be the case then it's fine, otherwise if this is likely to be a longer running feature it might be work moving as much as possible into a shared module. What struck me was the amount of model methods etc we were duplicating.